src/main/java/seedu/address/model/ingredient/Ingredient.java

```diff
@@ -58,8 +58,8 @@ public boolean equals(Object other) {
         }
 
         Ingredient otherIngredient = (Ingredient) other;
-        return otherIngredient.getName().equals(getName())
-                && otherIngredient.getQuantity().equals(getQuantity());
+        return otherIngredient.getName().equals(getName());
+        // && otherIngredient.getQuantity().equals(getQuantity());
```

Comment:
I have made changes to `UniqueIngredientList` and `CompatibleIngredientList` so that `Ingredient#equals()` will not be called in those classes any more, except in their own `equals()` method. I think that we should still check for equality of quantity so that when we are comparing between ingredient lists for example, this can be used to tell if the ingredient lists have the same items.

Reply:
bump, would be good to check for equality of quantity here too

Reply:
Will fix with next push